clients/fides-js/src/components/Overlay.tsx

```diff
@@ -153,7 +153,7 @@ const Overlay: FunctionComponent<Props> = ({
   }, [showBanner, setBannerIsOpen]);
 
   useEffect(() => {
-    if (!experience || options.modalLinkId === "") {
+    if (options.fidesEmbed || !experience || options.modalLinkId === "") {
```

Comment:
we don't want to be constantly polling for a modal link when the CMP is embedded in the page!